### PR TITLE
Create introduction.md

### PR DESCRIPTION
**Pull request by jonathanglassman**
### Context

Introduce the tech doc tool.

### Changes proposed in this pull request

Introduce the tech doc tool.

### CODE REVIEW

File: source/introduction/introduction.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+This tool is a template for technical and developer documentation. 
+
+It uses the [Middleman static site generator](https://middlemanapp.com/) [external link] to convert markdown files into HTML-based documentation that looks consistent with the GOV.UK and GDS style and format. 
+
+It is suitable for multiple styles of documentation, including API documentation. 
+
+## Why should you use this?
+
+It imposes consistency on technical documentation which should reassure end users.
```
Comment (bahmady) [inline source/introduction/introduction.md]: Only minor comment here: I'm not sure about using the word 'impose'

Comment (jonathanglassman) [reply source/introduction/introduction.md]: how about promote?

Comment (bahmady) [reply source/introduction/introduction.md]: > how about promote?

 💯

**Comment by bahmady** [comment]
@jonathanglassman no reason to withhold approval but just a small recommendation in the context of this PR's work: we could add a note saying that this project's documentation is actively under construction/still in beta etc.

**Pull request merged**